lib/utils.py

```diff
@@ -54,27 +55,27 @@ def make_request(
     server is not in sync with our NTP server or local computer).
     """
     # Ensure the URL is not malformed
-    site = site.replace("//", "/").lstrip("/")
+    site = re.sub(r"(?<!:)//+", "/", site).lstrip("/")
```

Comment:
Were you seeing any issues with the `site.replace` function that using `re.sub` solves?